process.md

```diff
@@ -0,0 +1,57 @@
+# Process
+
+We follow the OKR process, see the [2019/2020 Delivery Plan](https://wellcomecloud.sharepoint.com/sites/wc2/DE/RoadMaps/Delivery%20Plans/Delivery%20Plan%202019-20%20Digital%20Engagement.pdf?csf=1&e=WUG2dI&cid=56bdefa5-c5d4-44a1-99e7-96a6975d10fc)
+
+Delivery over process - do not follow these guidelines if they prevent you from delivering value as defined by our objectives.
+
+## Projects
+
+Projects are quarterly iterations of the projects [available in Github](https://github.com/wellcometrust/platform/projects).
+
+Issues are binned or recycled on a quarterly basis.
+
+## Pull Requests (PRs)
```

Comment:
I feel like (most of) these PR requirements might be more appropriate as ticket/issue requirements; if they only become visible when the PR is opened then that feels a bit too late to me - does that make sense? That is based on the assumption that usually tickets and PRs are one-to-one.